pass-deposit-services/deposit-core/src/main/resources/application.properties

```diff
@@ -20,11 +20,13 @@ pass.client.url=${PASS_CLIENT_URL:localhost:8080}
 pass.client.user=${PASS_CLIENT_USER:fakeuser}
 pass.client.password=${PASS_CLIENT_PASSWORD:fakepassword}
 
-ftp.host=${FTP_HOST:localhost}
-ftp.port=${FTP_PORT:21}
+ftp.host=${PMC_FTP_HOST:localhost}
+ftp.port=${PMC_FTP_PORT:21}
```

Comment:
I was expecting to see `pmc.ftp.user` and `pmc.ftp.password`

Reply:
When I add this prefix, I'm getting errors regarding the configuration. I'm not exactly sure what is causing this, but I will continue looking into this.

Reply:
When do you get the error?

Reply:
Done.